src/test/java/nl/tudelft/ewi/devhub/webtests/views/IssueView.java

```diff
@@ -0,0 +1,24 @@
+package nl.tudelft.ewi.devhub.webtests.views;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+
+public abstract class IssueView extends AuthenticatedView {
```

Comment:
What is exactly the philosophy behind this class? Can't this be an interface that views may implement? I am afraid that with this view we potentially hit an diamond problem with the view structure.